File: SistemaAcademicoUTPL/src/sistemaacademicoutpl/SistemaAcademicoUTPL.java
package sistemaacademicoutpl;
import java.util.Scanner;
import java.util.Random;
public class SistemaAcademicoUTPL {
   public static void main(String[] args) throws Exception {
        Random random = new Random();
        Scanner teclado= new Scanner(System.in);
        double notaACD, notaAPE, notaAA,total,porcentApp,porcentRep;
        String nombre,condicion;
        int ced,contApp=0,contRep=0;
        do {
            System.out.println("Ingresar el nombre del estudiante");
            nombre=teclado.next();
            System.out.println("Ingresar numero de cedula del estudiante");
            ced=teclado.nextInt();
            System.out.println("Notas del Bimestre");
            notaACD=random.nextInt(10)+1;
            notaAPE=random.nextInt(10)+1;
            notaAA=random.nextInt(10)+1;
            total=(notaACD+notaAPE+notaAA)/3;
            System.out.println("Nota Total: "+total);
            System.out.println("--------//REPORTE//--------");
            System.out.println("--NOTA DEL BIMESTRE--");
            System.out.println("Nota ACD: "+notaACD);
            System.out.println("Nota APE: "+notaAPE);
            System.out.println("Nota AA: "+notaAA);
            if (total>=7) {
                System.out.println("Nombre del estudiante: "+ nombre + " cedula: "+ced +"Aprovado");
                contApp++;
            }else{
                System.out.println("Nombre del estudiante: "+ nombre + " cedula: "+ced +"Reprovado");
                contRep++;
            }
            System.out.println("______________________________________");
            System.out.println("Desea ingresar mas estudiantes?");
            condicion=teclado.next().toLowerCase();
        } while (("s").equals(condicion));
        porcentApp=((contApp*100)/(contRep+contApp));
        porcentRep=((contRep*100)/(contRep+contApp));
        System.out.println("______________________________________________");
        System.out.println("El porcentaje de estudiantes aprovados es: "+porcentApp+"%");
        System.out.println("El porcentaje de estudiantes reprovados es: "+porcentRep+"%");
   }
}

File: SistemaAcademicoUTPL/SistemaAcademicoUTPL.py
import random

totalApro = 0
totalRep = 0
condicion = 's'

while condicion == 's':
    nombre = input("Ingresar el nombre del estudiante: ")
    ced = int(input("Ingresar numero de cedula del estudiante: "))
    nota_acd = random.randint(1, 10)
    nota_ape = random.randint(1, 10)
    nota_aa = random.randint(1, 10)
    total = (nota_acd + nota_ape + nota_aa) / 3
    print("--------//REPORTE//--------")
    print("--NOTA DEL BIMESTRE--")
    print("Nota ACD: ", nota_acd)
    print("Nota APE: ", nota_ape)
    print("Nota AA: ", nota_aa)
    print("Nota Total: ", total)
    if total >= 7:
        print("Nombre del estudiante: ", nombre, " cedula: ", ced, "Aprovado")
        totalApro += 1
    else:
        print("Nombre del estudiante: ", nombre, " cedula: ", ced, "Reprovado")
        totalRep += 1
    print("______________________________________")
    condicion = input("Desea ingresar mas estudiantes?").lower()

porcentajeApro = (totalApro * 100) / (totalApro + totalRep)
porcentajeRep = (totalRep * 100) / (totalApro + totalRep)
print("______________________________________________")
print("El porcentaje de estudiantes aprovados es: ", porcentajeApro,"%")
print("El porcentaje de estudiantes reprovados es: ", porcentajeRep,"%")